packages/cli-e2e/utils/browser.ts

```diff
@@ -115,7 +115,7 @@ export async function isElementClickable(page: Page, selector: string) {
       }
 
       // Edge before switching to Chromium
-      const isOldEdge = !!window.StyleMedia;
+      const isOldEdge = !!(window as any).StyleMedia;
```

Comment:
Missing some types ? 

FYI, it's relatively simple to augment global types like window and HTMLElement, if that's something you'd prefer:

```
declare global {

  interface Window {
    StyleMedia?: {}
  }

  interface HTMLElement {
       shadowRoot?: {}
   }

}
```